src/main/java/org/fundacionjala/coding/juan/eanValidator.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.juan;
+
+/**
+ * @author Juan Pablo
+ */
+public class eanValidator {
+    public static boolean validating(final String eanCode) {
+        int sum = 0;
+        int checksum = 0;
+        int finalNumbre = 12;
+        for(int i = 0; i < finalNumbre; i++) {
+            if(i%2 == 1) {
+                sum += ((eanCode.charAt(i)-48)*3);
```

Comment:
You should use the Character.getNumericValue for legibility and easy maintenance.